src/app/store/AuthModel.ts

```
@@ -484,107 +486,135 @@ export const getAuthModel = (): AuthModel => ({
   }),
   authFacebook: thunk(async (actions, options) => {
     return await new Promise<AuthPromiseResolveType>(async (resolve, reject) => {
-      const { declinedPermissions, isCancelled } = await LoginManager.logInWithPermissions([
-        "public_profile",
-        "email",
-      ])
-      if (declinedPermissions?.includes("email")) {
-        reject(
-          new AuthError("Please allow the use of email to continue.", "Email Permission Declined")
-        )
-      }
-      const accessToken = !isCancelled && (await AccessToken.getCurrentAccessToken())
-      if (!accessToken) {
-        return
-      }
-
-      const responseFacebookInfoCallback = async (error: any | null, result: any | null) => {
-        if (error) {
-          reject(new AuthError("Error fetching facebook data", error))
-          return
-        }
+      try {
+        const { declinedPermissions, isCancelled } = await LoginManager.logInWithPermissions([
```

the `loginWithPermissions` call can throw and this leads to login failing silently, wrap in a try catch instead